spark/hbase-spark/src/main/scala/org/apache/hadoop/hbase/spark/JavaHBaseContext.scala

```diff
@@ -244,19 +246,22 @@ class JavaHBaseContext(@transient val jsc: JavaSparkContext,
    *                      JavaRDD
    * @return              New JavaRDD that is created by the Get to HBase
    */
-  def bulkGet[T, U](tableName: TableName,
-                    batchSize: Integer,
-                    javaRdd: JavaRDD[T],
-                    makeGet: Function[T, Get],
-                    convertResult: Function[Result, U]): JavaRDD[U] = {
```

Comment:
in fact,  i feel the old way is much clearer to read...
but spotless... (shrug)